src/plugins/mer/meremulatordevice.h

```diff
@@ -101,6 +101,18 @@ class MerEmulatorDevice : public MerDevice
     void setSubnet(const QString& subnet);
     QString subnet() const;
 
+    void setMemorySize(int sizeMb);
+    int memorySize() const;
+
+    void setVdiInfo(const QString &vdiPath, int vdiCapacity, int vdiSizeOnDisk);
```

Comment:
It would be nice to use the Mb/Gb suffix as you did for setMemorySize. Ideally member variable, access function and/or it's parameter name would use the same words. `setMeMorySizeMb(memorySizeMb) { m_memorySizeMb = memorySizeMb; }`. I accept that it does not look nice for very long names but this is not the case and it makes code searching/manipulation easier.